### ISSUE TITLE
problem with the result of PDF/A-check

### ISSUE THREAD

**Issue by marc-schueller** [veraPDF/veraPDF-library#1186]
Dear Sir or Madam,

we use veraPDF (version 1.14.102, build date 2019-07-16T04:00:00+02:00) to validate pdf documents.
The resulting report contains the information that a real value within the pdf file is too big or too small. What might be the reason for that error?

The pdf file is the output of a programme. So this one might be responsible for the described issue. In a second step or work-around we have already tried to convertig the pdf file into a PDF/A compliant file with the help of another tool: PDF-XChange Editor Plus (version 9.0, build 354.0)
We have choosen the option "save file as... " with file type PDF/A-document (conformity with PDF/A-1b). While testing the converted file the same error occurred.

So our questions are:
Did veraPDF work correctly? (i. e.: Is the pdf-file really not compliant with PDF/A-1b?)
If so: what should be done in order to avoid this error?

If not: Is this issue (false-negative) already fixed in a higher version than we use?

We are really looking forward to hearing from you. Please maintain the confidentiality of the attached files.

-- 
with best regards
on behalf

Marc Schüller
____________________________________
unit A 2 / department III
Federal Information Technology Centre

[613-0027809-0003.pdf](https://github.com/veraPDF/veraPDF-library/files/7286186/613-0027809-0003.pdf)
[613-0027809-0003-A.pdf](https://github.com/veraPDF/veraPDF-library/files/7286188/613-0027809-0003-A.pdf)
[veraPDF_reportfile.txt](https://github.com/veraPDF/veraPDF-library/files/7286193/veraPDF_reportfile.txt)

**Comment by mkl-public** [comment]
>Did veraPDF work correctly? (i. e.: Is the pdf-file really not compliant with PDF/A-1b?)

Yes. E.g. the content stream of the first document page contains this instruction:

    175.438597 0 0 140.84507 -80605.102077 -34388.688859 cm

Clearly the absolut value of the last operand is larger than 32767 but the PDF/A-1 specification requires that the architectural limits in the based-upon PDF reference shall not be violated and those limits contain ±32767 as largest and smallest real values.

> what should be done in order to avoid this error?

Ask the producer of those wanna-be PDF/A-1 files to fix their program.
